mllib/src/main/scala/org/apache/spark/mllib/clustering/KMeans.scala

```diff
@@ -193,12 +207,19 @@ class KMeans private (
 
     val initStartTime = System.nanoTime()
 
-    val centers = if (initializationMode == KMeans.RANDOM) {
-      initRandom(data)
-    } else {
-      initKMeansParallel(data)
+    val centers = initialModel match {
+      case Some(kMeansCenters) => {
+        Array.tabulate(runs)(r => kMeansCenters.clusterCenters
```

Comment:
@jkbradley Is it ok to explicitly set the value 1 in the place of `runs` here or should we add a `require(runs==1, "can only run once")` in setInitialModel() ?